### ISSUE TITLE
Import error diagnostic is incomplete.

### ISSUE THREAD

**Issue by zong-zhe** [kcl-lang/kcl#271]
## Enhancement
At present, the error diagnostic output about import in KCL is as follows.

KCL:
```
import math as m // `mt` is written as `m` here by mistake.

two = mt.log10(100)
```

The diagnositc:
```
KCL Compile Error[E2L23] : A complie error occurs during compiling
---> File /KCLVM/test/grammar/import/builtin_import_as/main.k:3:7
3 |two = mt.log10(100)
       7 ^  -> Failure
name 'mt' is not defined
```

This doesn't help much for helping the user find where the typo.
Maybe we should learn rust.
It is more useful to output such diagnostic information.

```
KCL Compile Error[E2L23] : A complie error occurs during compiling
---> File /KCLVM/test/grammar/import/builtin_import_as/main.k:3:7
3 |two = mt.log10(100)
       7 ^  -> Failure
name 'mt' is not defined

help: consider importing one of these items
   |
0  |    import math as mt 
   |
0  |    import mt
```

**Comment by zong-zhe** [comment]
I couldn't be sure if the name not defined was the name in the import statement, so I added a suggestion, like https://github.com/kcl-lang/kcl/pull/715

**Comment by i-zhen** [comment]
Good choice

**Comment by Peefy** [comment]
Closed by #731
